cmd/csi-provisioner/csi-provisioner.go

```diff
@@ -193,14 +192,13 @@ func main() {
 		provisionerOptions = append(provisionerOptions, controller.AdditionalProvisionerNames([]string{supportsMigrationFromInTreePluginName}))
 	}
 
-	var scLister storagelistersv1.StorageClassLister
+	// Create informer to prevent hit the API server for all resource request
+	factory := informers.NewSharedInformerFactory(clientset, ctrl.ResyncPeriodOfCsiNodeInformer)
+	scLister := factory.Storage().V1().StorageClasses().Lister()
```

Comment:
Do we have a unit test that could cover this?

Reply:
Apparently not, this whole `main()` function isn't unit-tested.